docs/reference/core.md

```diff
@@ -17,3 +17,6 @@ functions.  Use at your own risk.
 
 ## Versions
 ::: synapseclient.core.version_check
+
+## Permissions
```

Comment:
Can we add this Permissions documentation to it's own page. I don't think this belongs in the `core.md` documentation.

So

1. Make a `permissions.md`
2. Move these lines over to that markdown
3. Add the permissions md reference in the `mkdocs.yml` nav section

Reply:
@BryanFauble Thanks for catching this. I must have missed the 'above core' part in your early comment. Will tweak this soon. Thanks